pushbullet_cli/app.py

```diff
@@ -65,7 +65,10 @@ def _push(data_type, title=None, message=None, channel=None, device=None, file_p
     elif data_type == "url":
         pb.push_link(title=title or url, url=url, **data)
     elif data_type == "text":
-        pb.push_note(title=title or "Note", **data)
```

Comment:
I prefer to keep it in one line. Please change it to `title if title is not None else "Note"`